Gems/Multiplayer/Code/Include/Multiplayer/Components/MultiplayerComponentRegistry.h

```diff
@@ -67,11 +67,26 @@ namespace Multiplayer
         //! @return reference to the requested component data, an empty container will be returned if the NetComponentId does not exist
         const ComponentData& GetMultiplayerComponentData(NetComponentId netComponentId) const;
 
+        //! Returns the combined hashes of all the multiplayer components creating a single system version hash that can be quickly compared between the server and client app.
+        //! @return a 64-bit system-wide hash value representing of all of the multiplayer components
+        AZ::HashValue64 GetSystemVersionHash() const;
```

Comment:
nit: GetMultiplayerSystemVersionHash or GetCombinedMultiplayerVersionHash? 

Feel that GetSystemVersionHash is a little too brief and not as self-describing as we could make it

Reply:
I would say GetMultiplayerSystemVersionHash between the two. Personally I find the namespace and class qualifies GetSystemVersionHash reasonably well enough.

Reply:
Thanks for the feedback. Totally, both of the same thoughts crossed my mind, but I went with the less redundant "GetSystemVersion". The namespace and class covers the base nicely. Plus the calls for an outsider look like...
_GetMultiplayerComponentRegistry()->GetSystemVersionHash()_